Example/HorizonCalendarExample/HorizonCalendarExample/SwiftUIDayView.swift

```diff
@@ -0,0 +1,65 @@
+// Created by Bryan Keller on 8/23/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import HorizonCalendar
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct SwiftUIDayView: View {
+
+  let dayNumber: Int
+  let isSelected: Bool
+
+  var body: some View {
+    ZStack(alignment: .center) {
+      if #available(iOS 15.0, *) {
+        Circle()
+          .strokeBorder(isSelected ? Color(UIColor.blue) : .clear, lineWidth: 1)
+          .background {
+            Circle()
+              .foregroundColor(
+                (isSelected ? Color(UIColor.blue.withAlphaComponent(0.15)) : .clear))
+          }
+          .aspectRatio(1, contentMode: .fill)
+      } else {
+        Circle()
+          .foregroundColor(
+            (isSelected ? Color(UIColor.blue.withAlphaComponent(0.15)) : .clear))
+          .aspectRatio(1, contentMode: .fill)
+      }
+      Text("\(dayNumber)").foregroundColor(Color(UIColor.label))
+    }
+  }
+
+}
+
+@available(iOS 13.0.0, *)
```

Comment:
I'd assume we can bump the deployment target in the example app above iOS 13 so we don't need these? (even if we keep it lower in the library)

Reply:
That simplifies a few things, thanks for noticing this 😄